dashboards/Secrets/Vault/Vault-Summary.json

```diff
@@ -790,7 +822,7 @@
     ]
   },
   "timezone": "utc",
-  "title": "Secrets: Summary Vault",
+  "title": "/dcos-generic/dashboards/Secrets/Vault/Vault-Summary",
```

Comment:
Can we change this back to the standardized title format?

Reply:
The only test I did was to change the IDs and re-import manually the dashboards on the {{soak113s}} cluster from where they have been exported.